src/fable-fsharp/Replacements.fs

```diff
@@ -190,6 +193,8 @@ module Util =
                     |> makeCall com i.range (Fable.PrimitiveType (Fable.Number Int32))
                 Fable.Apply(op, [comp; makeConst 0], Fable.ApplyMeth, i.returnType, i.range)
                 |> Some
+            | Fable.Class None ->  // For enums only ("DeclaredType <enumName> Class null")
+                Fable.Apply(op, args, Fable.ApplyMeth, i.returnType, i.range) |> Some  
```

Comment:
Sorry, I just read your comment in the issue report and I understand now why this is needed. Hmm, at the end I think we have to add enums to Fable's AST. It shouldn't be difficult though, I'll do it :) It's wonderful that you caught that with your tests :+1: